### DISCUSSION TITLE
Private key import

### DISCUSSION THREAD

**Discussion by Deimon5762** [Chia-Network/chia-blockchain#17459] [Support]
Hello, please tell me how to import a private key into the Chia blockchain wallet. I can't regain access to my Chia coins because I stored them in the Arbor wallet. The support for Arbor wallet has now ceased, and it no longer functions. The private key is my last hope because the seed phrases generated in the Arbor wallet do not work with other wallets.

**Comment by BrandtH22** [comment]
Hey @Deimon5762 , when importing mnemonics into the chia client there is an option to "Import from 12 Word Mnemonic". That is likely the option you will want to use and make sure you have the capitalization of words correct.

You will need to use the mnemonics to recover the wallet as the private keys do not have the necessary information.

As a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
